presto-docs/src/main/sphinx/release/release-0.249.2.rst

```diff
@@ -0,0 +1,7 @@
+===============
+Release 0.249.2
+===============
+
+General Changes
+---------------
+* Add flag to enable undo for alterPartition operation.
```

Comment:
1. this should be under "Hive Changes" .  
2.  The property applies to all metastore operations, and not just alterPartition.  Also, should specify the name of the property and its default value.   Something like

```
Add configuration property ``hive.undo-metastore-operations-enabled`` to set whether to undo metastore operations after encountering an error.  This flag is true by default, which maintains the previous behavior.
```